app/src/main/java/com/xmartlabs/template/helper/OnBoardingValidator.kt

```diff
@@ -0,0 +1,16 @@
+package com.xmartlabs.template.helper
+
+object OnBoardingValidator {
```

Comment:
should be `OnboardingValidator` as `Onboarding` is only one word

Reply:
i don't think an object is needed here maybe just a file?